lib/Service/PlayerActionService.php

```diff
@@ -78,35 +77,56 @@ private function getConfig()
      */
     public function sendAction($displays, $action)
     {
-        if (!$this->triggerPlayerActions)
+        if (!$this->triggerPlayerActions) {
             return;
+        }
 
         // XMR network address
-        if ($this->xmrAddress == null)
+        if ($this->xmrAddress == null) {
             $this->xmrAddress = $this->getConfig()->getSetting('XMR_ADDRESS');
+        }
 
-        if (!is_array($displays))
+        if (!is_array($displays)) {
             $displays = [$displays];
+        }
 
         // Check ZMQ
-        if (!Environment::checkZmq())
-            throw new ConfigurationException(__('ZeroMQ is required to send Player Actions. Please check your configuration.'));
+        if (!Environment::checkZmq()) {
+            throw new ConfigurationException(
+                __('ZeroMQ is required to send Player Actions. Please check your configuration.')
+            );
+        }
 
-        if ($this->xmrAddress == '')
+        if ($this->xmrAddress == '') {
             throw new InvalidArgumentException(__('XMR address is not set'), 'xmrAddress');
+        }
 
         // Send a message to all displays
         foreach ($displays as $display) {
             /* @var Display $display */
-            if ($display->xmrChannel == '' || $display->xmrPubKey == '')
-                throw new InvalidArgumentException(__('This Player is not configured or ready to receive push commands over XMR. Please contact your administrator.'), 'xmrRegistered');
+            if ($display->xmrChannel == '' || $display->xmrPubKey == '') {
+                throw new InvalidArgumentException(
+                    __(sprintf(
```

Comment:
This needs to be the other way around `sprintf(__())`

Reply:
whoops, fixed